libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -454,19 +424,7 @@ enum BreezSDKMapper {
         }
         let nodeConfig = try asNodeConfig(nodeConfig: nodeConfigTmp)
 
-        return Config(
-            breezserver: breezserver,
-            chainnotifierUrl: chainnotifierUrl,
-            mempoolspaceUrl: mempoolspaceUrl,
-            workingDir: workingDir,
-            network: network,
-            paymentTimeoutSec: paymentTimeoutSec,
-            defaultLspId: defaultLspId,
-            apiKey: apiKey,
-            maxfeePercent: maxfeePercent,
-            exemptfeeMsat: exemptfeeMsat,
-            nodeConfig: nodeConfig
-        )
+        return Config(breezserver: breezserver, chainnotifierUrl: chainnotifierUrl, mempoolspaceUrl: mempoolspaceUrl, workingDir: workingDir, network: network, paymentTimeoutSec: paymentTimeoutSec, defaultLspId: defaultLspId, apiKey: apiKey, maxfeePercent: maxfeePercent, exemptfeeMsat: exemptfeeMsat, nodeConfig: nodeConfig)
```

Comment:
Are these formatting changes intended? There are many others like this one

Reply:
Kind of, they now match the Nodeless codegen formatting